### PR TITLE
Add FreemodePed, SynchronizedScene, and enum docs

### PR DESCRIPTION
**Pull request by furaihan**

### CODE REVIEW

**Review by NightYield — APPROVED** [review]

File: Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs [hunk 1]
```diff
@@ -13,10 +13,24 @@ public static partial class NativeWrappers
         /// <remarks>
         /// A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
         /// </remarks>
-        public static void TaskStartScenarioInPlace(Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
+        public static void TaskStartScenarioInPlace(Ped ped, string scenario, int unkDelay = 0, bool playEnterAnimation = true)
```
Comment (Rich-Dunne) [inline Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs]: Why did you change this from the Scenario enum?
```suggestion
        public static void TaskStartScenarioInPlace(Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
```

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs [hunk 2]
```diff
@@ -23,14 +23,51 @@ public static class PedExtensions
         public static void ShootFromVehicle(this Ped ped, Ped target, float unknown = 0) => NativeWrappers.TaskVehicleShootAtPed(ped, target, unknown);
 
         /// <summary>
-        /// Causes this ped to start playing the specified scenario.
+        /// Causes this ped to start playing the specified scenario in place.
         /// </summary>
-        public static void StartScenarioInPlace(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenario, unkDelay, playEnterAnimation);
-
+        public static void StartScenario(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
```
Comment (Rich-Dunne) [inline Source/RageCommunity.Library/Extensions/PedExtensions.cs]: Why was this method name changed?
```suggestion
        public static void StartScenarioInPlace(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
```

**Review by Rich-Dunne — COMMENTED** [review]
Thanks for adding summary description to the ped models, that could be helpful.  Is there a benefit to using the hex hash over the numerical hash?  Seems like you went through a lot of extra work to convert all those.  I also had a couple questions regarding some minor changes, if you wouldn't mind addressing those.

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs (hunk 2 above)
Comment (furaihan) [reply Source/RageCommunity.Library/Extensions/PedExtensions.cs]: Because I added an overload to play scenario at the specified position. but I'm fine if you change it back to the original

File: Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs (hunk 1 above)
Comment (furaihan) [reply Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs]: because the [natives](https://nativedb.dotindustries.dev/natives/0x142A02425FF02BD9) says that this parameter is a string

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs [hunk 3]
```diff
@@ -23,14 +23,51 @@ public static class PedExtensions
...
+        /// Causes this ped to start playing the specified scenario at the given <paramref name="position"/>
+        /// </summary>
+        /// <param name="ped">The target <see cref="Ped"/></param>
+        /// <param name="scenario">The scenario to be played</param>
+        /// <param name="position">The position that the scenario will be playing at</param>
+        /// <param name="heading">The heading</param>
+        /// <param name="duration">in miliseconds, if <c>-1</c> the task will never timeout</param>
+        /// <param name="sittingScenario"><c>true</c> if the given <paramref name="scenario"/> is a sitting scenario, otherwise <c>false</c></param>
+        /// <param name="teleport">if set to <c>true</c> the <paramref name="ped"/> will be teleported at the given <paramref name="position"/></param>
+        public static void StartScenario(this Ped ped, Scenario scenario, Vector3 position, float heading, int duration, bool sittingScenario, bool teleport)
```
Comment (Rich-Dunne) [inline Source/RageCommunity.Library/Extensions/PedExtensions.cs]:
```suggestion
        public static void StartScenarioAtPosition(this Ped ped, Scenario scenario, Vector3 position, float heading, int duration, bool sittingScenario, bool teleport)
```

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs (hunk 2 above)
Comment (Rich-Dunne) [reply Source/RageCommunity.Library/Extensions/PedExtensions.cs]: I see the other method, and suggested a change to that name as well for the sake of clarity and consistency with our naming conventions.  Thanks!

File: Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs [hunk 4]
```diff
@@ -13,10 +13,24 @@ public static partial class NativeWrappers
...
         {
             NativeFunction.Natives.TASK_START_SCENARIO_IN_PLACE(ped, scenario, unkDelay, playEnterAnimation);
```
Comment (Rich-Dunne) [inline Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs]:
```suggestion
            NativeFunction.Natives.TASK_START_SCENARIO_IN_PLACE(ped, scenario.ToString(), unkDelay, playEnterAnimation);
```

File: Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs (hunk 1 above)
Comment (Rich-Dunne) [reply Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs]: I added a change suggestion to convert the parameter to a string on the native this wrapper points to.  Using the enum in the wrapper can help the user through intellisense.

File: Source/RageCommunity.Library/Extensions/PedExtensions.cs [hunk 5]
```diff
@@ -23,14 +23,51 @@ public static class PedExtensions
...
 
         /// <summary>
-        /// Causes this ped to start playing the specified scenario.
+        /// Causes this ped to start playing the specified scenario in place.
         /// </summary>
-        public static void StartScenarioInPlace(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true) => NativeWrappers.TaskStartScenarioInPlace(ped, scenario, unkDelay, playEnterAnimation);
-
+        public static void StartScenarioInPlace(this Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
+        {
+            NativeWrappers.TaskStartScenarioInPlace(ped, scenario.ToString(), unkDelay, playEnterAnimation);
```
Comment (Rich-Dunne) [inline Source/RageCommunity.Library/Extensions/PedExtensions.cs]:
```suggestion
            NativeWrappers.TaskStartScenarioInPlace(ped, scenario, unkDelay, playEnterAnimation);
```

**Review by Rich-Dunne — COMMENTED** [review]
Fixing an oopsie
